roles/openshift_openstack/templates/heat_stack_server.yaml.j2

```diff
@@ -215,6 +226,21 @@ resources:
 {% endif %}
       scheduler_hints: { get_param: scheduler_hints }
 
+  lb_pool_member:
+    condition: attach_to_lb
+    type: OS::Neutron::LBaaS::PoolMember
+    properties:
+      address:
+        get_attr:
+          - server
+          - addresses
+          - { get_param: net_name }
+          - 0
```

Comment:
shall we place a FIXME for multiple private networks support?

Reply:
done